doc/oper.xml

```diff
@@ -1387,3 +1387,34 @@ gap> DigraphShortestDistance(gr, [1, 3], [3, 5]);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="PartialOrderDigraphMeetOfVertices">
+<ManSection>
+  <Oper Name="PartialOrderDigraphMeetOfVertices" Label="for a digraph and two vertices"
+        Arg="digraph, u, v"/>
+  <Oper Name="PartialOrderDigraphJoinOfVertices" Label="for a digraph and two vertices"
+        Arg="digraph, u, v"/>
+  <Returns>An integer</Returns>
+    <Description>
+    If the first argument is a partial order digraph <Ref Prop="IsPartialOrderDigraph"/>
+    then the function returns the meet, or the join, of the two input vertices. 
+    The meet (or join) is guarenteed to exist and be unique when the first
+    argument satisfies <Ref Prop="IsMeetSemilatticeDigraph"/> 
+    (or <Ref Prop="IsJoinSemilatticeDigraph"/>) - see the documentation for these
+    properties for the definition of the meet (or the join).
+
```

Comment:
The documentation should note that, in the case that the meet/join does not exist, the operation returns `<K>fail</K>`.